mlrun/feature_store/retrieval/base.py

```diff
@@ -614,8 +609,41 @@ def _build_relation(
                 linked_list_relation.head.data["save_index"] = keys
             return linked_list_relation
 
+        def _build_entity_rows_relation(entity_rows_relation, fs_name, fs_order):
+            feature_set_in_entity_list = feature_set_entity_list_dict[fs_name]
+            feature_set_in_entity_list_names = list(feature_set_in_entity_list.keys())
```

Comment:
Wow, these are confusing names... It sounds like a bool indicating whether the fset is in the entity list...
Since this function only has a single fset as input, why not just call it `feature_set_entity_list` and `feature_set_entity_names`?